src/index.js

```diff
@@ -98,17 +100,19 @@ class Kalendaryo extends Component {
   getWeeksInMonth = (date = this.state.date) => {
```

Comment:
I think it would be a good convenience to have the user to be able invoke this function like so:

* `getWeeksInMonth()` - Returns the weeks in the month of the current [`#date`](#date) state, with the week starting at the value of the [`#firstDayOfWeek`](#firstdayofweek) prop
* `getWeeksInMonth(date)` - Returns the weeks in the month of the given `date` argument, with the week starting at the value of the [`#firstDayOfWeek`](#firstdayofweek) prop
* `getWeeksInMonth(number)` - Returns the weeks in the month of the current [`#date`](#date) state, with the week starting at the given `number` argument
* `getWeeksInMonth(date, number)` - Returns the weeks in the month of the given `date` argument, with the week starting at the given `number` argument

See [`#getFormattedDate`](https://github.com/geeofree/kalendaryo/blob/master/src/index.js#L45-L59), [`#getDateNextMonth`](https://github.com/geeofree/kalendaryo/blob/master/src/index.js#L61-L75), [`#getDatePrevMonth`](https://github.com/geeofree/kalendaryo/blob/master/src/index.js#L77-L91) for examples